lean4/src/putnam_1963_a6.lean

```diff
@@ -4,11 +4,11 @@ open BigOperators
 open Topology Filter
 
 theorem putnam_1963_a6
-(F1 F2 U V A B C D P Q : Fin 2 → ℝ)
+(F1 F2 U V A B C D P Q : EuclideanSpace ℝ (Fin 2))
 (r : ℝ)
-(E : Set (Fin 2 → ℝ) := {H : Fin 2 → ℝ | Euclidean.dist F1 H + Euclidean.dist F2 H = r})
-(M : Fin 2 → ℝ := midpoint ℝ U V)
-(hr : r > Euclidean.dist F1 F2)
+(E : Set (Fin 2 → ℝ) := {H : EuclideanSpace ℝ (Fin 2) | dist F1 H + dist F2 H = r})
```

Comment:
`Fin 2 → ℝ` here is the wrong type, it should also be `EuclideanSpace ℝ (Fin 2)`

Reply:
Good catch! Should be fixed in incoming PR.